graphql/src/store/resolver.rs

```diff
@@ -141,6 +155,45 @@ impl StoreResolver {
             }
         }
     }
+
+    fn handle_meta(
+        &self,
+        prefetched_object: Option<q::Value>,
+        object_type: &ObjectOrInterface<'_>,
+    ) -> Result<(Option<q::Value>, Option<q::Value>), QueryExecutionError> {
+        // Pretend that the whole `_meta` field was loaded by prefetch. Eager
+        // loading this is ok until we add more information to this field
+        // that would force us to query the database; when that happens, we
+        // need to switch to loading on demand
```

Comment:
Very soon I'll need to query the DB for subgraph errors for the meta field, where in the code would I do that? Do sneak a DB query into this function? Or does it go in the real prefetch code?

Reply:
That's the question I was trying to sidestep here - the thing is that we should avoid a database query when the user does not ask for fields that need database access. But it seems heinously hard to track robustly what fields are being accessed just based on the arguments that `resolve_object` gets - maybe for what you need you can have a special GraphQL type that you can use to trigger the query, similar to how the `MetaType` triggers this piece of code. I wouldn't bother putting it into prefetch, and just do the query straight from the resolver.

Reply:
I see, it needs to be a resolver on the field so not something we do here.